app/assets/javascripts/application.js

```diff
@@ -2,6 +2,7 @@
 //= require turbolinks
 //= require i18n
 //= require jquery.ui.autocomplete
+//= require jquery.ui.spinner
```

Comment:
patternfly spinner didn't allow me to customize the output and parsing - note that gb/mb is part of the input so user can type "5gb" and it will get parsed correctly